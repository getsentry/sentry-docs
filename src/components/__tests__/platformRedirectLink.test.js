import React from "react";
import renderer from "react-test-renderer";

import PlatformRedirectLink from "../platformRedirectLink";

describe("PlatformRedirectLink", () => {
  it("renders with to", () => {
    const tree = renderer
      .create(<PlatformRedirectLink to="/enriching-error-data/" />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it("renders without to", () => {
    const tree = renderer.create(<PlatformRedirectLink />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
