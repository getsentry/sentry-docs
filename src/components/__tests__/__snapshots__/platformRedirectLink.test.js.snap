// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlatformRedirectLink renders with to 1`] = `
<a
  className=""
  href="/platform-redirect/?next=%2Fenriching-error-data%2F"
>
  /platform-redirect/?next=%2Fenriching-error-data%2F
</a>
`;

exports[`PlatformRedirectLink renders without to 1`] = `
<a
  className=""
  href="/platform-redirect/"
>
  /platform-redirect/
</a>
`;
